src/app/Models/Product.ts

```diff
@@ -0,0 +1,16 @@
+export class Product {
```

Comment:
Since you are not creating an instance of this class, I would suggest that you can use it as an interface instead, otherwise you should add a constructor to this class in order to create a real model